cmd/liqo-controller-manager/main.go

```diff
@@ -339,10 +349,25 @@ func main() {
 	var wg = &sync.WaitGroup{}
 	broadcaster.StartBroadcaster(ctx, wg)
 
-	klog.Info("starting manager as controller manager")
-	if err := mgr.Start(ctx); err != nil {
-		klog.Error(err)
-		os.Exit(1)
+	wg.Add(1)
```

Comment:
Out of curiosity: Why do we need to add the following logic?

Reply:
We need to start the manager in a separate goroutine, and then take the client and forward it to the storage provisioner